### PR TITLE
docs(optimize): document kpi modal

### PR DESCRIPTION
**Pull request by OmranAbazid**
related to [#12374](https://github.com/camunda/camunda-optimize/issues/12374)

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by misiekhardcore — APPROVED** [review]

File: optimize/components/userguide/process-KPIs.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+id: process-KPIs
+title: Process KPIs
+description: "Learn how key performance indicators (KPIs) can be created and used in Optimize."
+---
+
+Key performance indicators (KPIs) are reports represented by single numerical values with predefined targets.
+To create a custom KPI, follow the steps below:
+
+1. On the right side of the **Home** page, select **Create New > New Report > Process KPI**.
```
Comment (christinaausley) [inline optimize/components/userguide/process-KPIs.md:10]: Do we mean of the **Collections** page?

Comment (christinaausley) [reply optimize/components/userguide/process-KPIs.md:10]: And **Create new > Report**? Maybe you can just direct me to the demo env 😅

Comment (misiekhardcore) [reply optimize/components/userguide/process-KPIs.md:10]: @christinaausley you are right, this is a **Collections** page right now. It used to be named **Home** some longer time ago but neither it's called **Home**, nor it is a default page of optimize. Good catch!

**Review by christinaausley — CHANGES_REQUESTED** [review]
Just need to adjust UI references 👍

**Review by christinaausley — APPROVED** [review]

**Comment by OmranAbazid** [comment]
Just updated a screenshot

**Review by christinaausley — APPROVED** [review]

**Comment by christinaausley** [comment]
Feel free to merge when you're ready 👍

**Pull request merged**